static/data/layers.js

```diff
@@ -0,0 +1,718 @@
+export default {
+    "version":8,
+    "metadata":{
+        "maputnik:renderer":"mbgljs"
+    },
+    "sources":{
+        "osmTiles":{
+            "type":"raster",
+            "tiles":[
+                "https://a.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                "https://b.tile.openstreetmap.org/{z}/{x}/{y}.png",
+                "https://c.tile.openstreetmap.org/{z}/{x}/{y}.png"
+            ],
+            "tileSize":256,
+            "maxzoom":19,
+            "paint":{
+                "raster-fade-duration":100
+            },
+            "attribution":"data © <a target=\"_top\" rel=\"noopener\" href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors."
+        },
+        "mediaWikiTiles":{
+            "type":"raster",
+            "tiles":[
+                "https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png?lang=uk"
+            ],
+            "tileSize":256,
+            "maxzoom":19,
+            "paint":{
+                "raster-fade-duration":100
+            },
+            "attribution":"map: <a target=\"_top\" rel=\"noopener\" href=\"https://foundation.wikimedia.org/wiki/Maps_Terms_of_Use\">WikiMedia</a>; data: © <a target=\"_top\" rel=\"noopener\" href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors."
+        },
+        "osmData":{
+            "type":"geojson",
+            "data":"https://dopomoha.pl/data/osm_data.geojson",
+            "maxzoom":12
+        },
+        "charityDropOff":{
+            "type":"geojson",
+            "data":"https://dopomoha.pl/data/zbiorki.geojson",
```

Comment:
Same here, can we revert to using local files, so we can iterate and develop locally?

Reply:
While osm data file uses osm as source and can be built locally this file is gathered manually by volounteers, in the future it will be generated from CMS that is being built and the file would be updated often enough that I don't think it would make sense to keep it in the repo so the best way is to serve it from the web